os/kernel/pthread/pthread_create.c

```diff
@@ -455,6 +455,11 @@ int pthread_create(FAR pthread_t *thread, FAR const pthread_attr_t *attr, pthrea
 	}
 
 	if (ret == OK) {
+#if defined(CONFIG_BINARY_MANAGER) && defined(CONFIG_APP_BINARY_SEPARATION)
+		/* Add tcb to binary thread list */
+
+		binary_manager_add_binlist(&ptcb->cmn);
```

Comment:
In task_create, add list is called before `task_activate`.
What is the difference?
Is it possible to move this before `task_activate` or move from `task_create`?

Reply:
modified